controllers/about/index.js

```diff
@@ -1,11 +1,35 @@
 'use strict';
+const path = require('path');
 const express = require('express');
 
-const { basicContent, flexibleContent } = require('../common');
+const contentApi = require('../../common/content-api');
+const { setCommonLocals } = require('../../common/inject-content');
+const { basicContent } = require('../common');
 
 const router = express.Router();
 
-router.get('/', flexibleContent());
+router.get('/', async function (req, res, next) {
+    try {
+        const entry = await contentApi.getFlexibleContent({
```

Comment:
This misleadingly named `getFlexibleContent` is only ever used on the about single. Inlined the code here. Likely another task on the CMS to either scope this API method to the about single or migrate to the newer flexible content type.